tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/MongoDb/IBsonElement.cs

```diff
@@ -0,0 +1,27 @@
+// <copyright file="IBsonElement.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.ComponentModel;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.MongoDb
+{
+    /// <summary>
+    /// MongoDB.Bson.BsonDocument interface for duck-typing
+    /// </summary>
+    [Browsable(false)]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public interface IBsonElement
```

Comment:
I think this could be a `[DuckCopy]` struct?

Reply:
I think you're right. Done in 46794d772bf3ede266cdd4202f691014f1a5eb75